apps/web/src/utils/css.ts

```diff
@@ -17,14 +17,14 @@ You should have received a copy of the GNU General Public License
 along with this program.  If not, see <http://www.gnu.org/licenses/>.
 */
 
-export function removeCss(id) {
-  var link = document.getElementById(id);
-  link.remove();
+export function removeCss(id: string) {
+  const link = document.getElementById(id);
+  link ? link.remove() : undefined;
```

Comment:
A simple if statement will suffice here.

Reply:
@Abdulrehman-Jafer

Reply:
Ok I'll remove the ternary..